nBTC/sources/nbtc.move

```diff
@@ -524,17 +501,25 @@ public fun request_signature_for_input(
 ) {
     // TODO: refactor this code, current struct is fix object ownership error model
     let (sign_id, sign_hash) = {
```

Comment:
Maybe we can move it to a helper function? Otherwise, no need for an inner block